qiskit/compiler/assemble.py

```diff
@@ -266,10 +268,15 @@ def _parse_pulse_args(backend, qubit_lo_freq, meas_lo_freq, qubit_lo_range,
     qubit_lo_range = qubit_lo_range or getattr(backend_config, 'qubit_lo_range', None)
     meas_lo_range = meas_lo_range or getattr(backend_config, 'meas_lo_range', None)
 
-    rep_time = rep_time or getattr(backend_config, 'rep_times', None)
+    if not rep_time and hasattr(backend_default, 'rep_times'):
```

Comment:
This can also remain as the original `rep_time = rep_time or getattr(backend_config, 'rep_times', None)` because `rep_time` is part of `BackendConfiguration` not `BackendDefaults`

Reply:
Ok, now I understand it better - I was a bit confused at this as well. Thanks!